src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ConfiguredProjectImplicitActivationTracking.cs

```diff
@@ -23,7 +23,7 @@ internal class ConfiguredProjectImplicitActivationTracking : OnceInitializedOnce
         private readonly IActiveConfigurationGroupService _activeConfigurationGroupService;
         private readonly ITargetBlock<IProjectVersionedValue<IConfigurationGroup<ProjectConfiguration>>> _targetBlock;
         private TaskCompletionSource<object> _isImplicitlyActiveSource = new TaskCompletionSource<object>();
```

Comment:
>TaskCompletionSource<object> _isImplicitlyActiveSource [](start = 16, length = 54)

Should this be `TaskCompletionSource<object?>`, to allow `TrySetResult(null)`?

Reply:
Yes, done.  Thanks!